pkg/core/fixtures/valid.yaml

```diff
@@ -0,0 +1,8 @@
+version: 0.1
+istio:
+  - istio-crds
```

Comment:
nit: is this really valid, or just good enough pass the basic validation

Reply:
I suspect it's valid, but the intention was that it was just enough to test the manifest parsing.